src/pyAEATsii/mapping.py

```diff
@@ -92,6 +79,20 @@ def _description(self, invoice):
             else _FIRST_SEMESTER_RECORD_DESCRIPTION
         )
 
+    def build_query_filter(self, year=None, period=None, last_invoice=None):
+        # TODO: IDFactura, Contraparte,
+        # FechaPresentacion, FechaCuadre, FacturaModificada,
+        # EstadoCuadre
+        result = {
+            'PeriodoLiquidacion': {
+                'Ejercicio': year,
+                'Periodo': _format_period(period),
+                }
+            }
+        if last_invoice:
+            result['ClavePaginacion'] = self._build_invoice_id(last_invoice)
```

Comment:
According to the wsdl, for both issued and recieved invoices the type of the object here is different from the invoice id in the 'submit' services. Specifically, here a `NumSerieFacturaEmisorResumenFin` attribute won't be accepted so we are potentially breaking the webservice contract if `last_invoice` turns out to be a grouped invoice.

I'd suggest to build this object in its own method. Maybe `_build_BC_inoice_id`? I don't know what the heck BC stands for but its part of the SII wsdl namespace regarding this object. And then make the old `_build_invoice_id` call it and add the additional atribute.

But regardless of the implementation details I wouldn't approve the PR with the corresponding unit test!

Pardon my delay but I felt this PR deserved an in-depth look.